core/controllers/base.py

```diff
@@ -70,6 +70,27 @@ def get(self):
         auth_services.establish_auth_session(self.request, self.response)
 
 
+class SessionEndHandler(webapp2.RequestHandler):
+    """Class which handles the creation of a new authentication session."""
```

Comment:
Perhaps for consistency: ``Handler for creating new authentication sessions.``

That being said, is this & the documentation below correct? Shouldn't this be indicating that it's ending a session?

Reply:
Done.